src/augtool.c

```diff
@@ -25,17 +25,18 @@
 #include "internal.h"
 #include "safe-alloc.h"
 
-#include <readline/readline.h>
-#include <readline/history.h>
 #include <argz.h>
+#include <ctype.h>
 #include <getopt.h>
 #include <limits.h>
-#include <ctype.h>
 #include <locale.h>
-#include <sys/types.h>
-#include <sys/stat.h>
 #include <pwd.h>
+#include <readline/history.h>
+#include <readline/readline.h>
+#include <signal.h>
 #include <stdarg.h>
+#include <sys/stat.h>
+#include <sys/types.h>
```

Comment:
Why did all the includes get reordered ? That makes it hard to spot which includes were added. Could you try and restore the old order ?